cmake/CTestConfig.cmake.in

```diff
@@ -0,0 +1,91 @@
+#----------------------------------------------------------------------------------------#
+#
+#   CTestConfig.cmake template for Kokkos
+#
+#----------------------------------------------------------------------------------------#
+
+#
+#   dash-board related
+#
+set(CTEST_PROJECT_NAME "Kokkos")
+set(CTEST_NIGHTLY_START_TIME "01:00:00 UTC")
+set(CTEST_DROP_METHOD "https")
+set(CTEST_DROP_SITE "cdash.nersc.gov")
+set(CTEST_DROP_LOCATION "/submit.php?project=${CTEST_PROJECT_NAME}")
+set(CTEST_CDASH_VERSION "1.6")
+set(CTEST_CDASH_QUERY_VERSION TRUE)
+set(CTEST_SUBMIT_RETRY_COUNT "1")
+set(CTEST_SUBMIT_RETRY_DELAY "30")
+
+#
+#   configure/build related
+#
+set(CTEST_BUILD_NAME "@BUILD_NAME@")
+set(CTEST_MODEL "@MODEL@")
+set(CTEST_SITE "@SITE@")
+set(CTEST_CONFIGURATION_TYPE "@BUILD_TYPE@")
+set(CTEST_SOURCE_DIRECTORY "@SOURCE_REALDIR@")
+set(CTEST_BINARY_DIRECTORY "@BINARY_REALDIR@")
+
+#
+#   configure/build related
+#
+set(CTEST_UPDATE_TYPE "git")
+set(CTEST_UPDATE_VERSION_ONLY ON)
```

Comment:
This makes ctest only use git to discover the version that is being tested which is what we want.
(Commenting because I had to look it up and I figured others might wonder too)